code/billy/jackelope_lab.py

```diff
@@ -0,0 +1,58 @@
+'''
+program: jackelope_lab 
+author: billy frick
+date: 20 september 2022
+
+function:   > this program will use a list to represent a jackelope population.
+            > the list will gain 2 new ints (newborns) for the jackelopes between the ages of 4 and 8.
+            > every jackelope that reaches 10 will be removed from the list (via .pop())
+            > years will be tracked for each instance of the while loop.
+            > the current population and years taken to reach it will be printed.
+'''
+
+# this will store the amount of years that have passed
+years_passed = 0
+
+# bonus variable to keep track of how many jackelopes passed away.
+# it seems as though 1 jackelope dies in a single year that passes.
+jackelope_deaths = 0
+
+# initial jackelope population is 2, and this is the list containing 2 ints to represent their age.
+jackalopes_population = [0,0]
+
+# this loop will continue until the jackelope population reaches 1000 (or greater).
+while len(jackalopes_population) < 1000:
+
+    # for each jackelope that is between 4 and 8, 2 more jackelopes will be added to the population list
+    for jackalope in jackalopes_population:
+
+        if 4 <= jackalope <= 8:
+
+            # adds 2 additional jackelope for each mating instance. Start at 0 to represent them being newborns.
+            jackalopes_population.append(0)
```

Comment:
It's interesting how much one line of code can make a difference. It seems the fix from last comment was to remove one of the `jackalopes_population.append(0)` lines. At least that's the diffirence I see.